lib/rucio/daemons/rsedecommissioner/profiles/__init__.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Decommissioning profile definitions."""
+
+import logging
+from collections.abc import Callable, Iterable
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any
+
+
+def get_profile(name: str):
```

Comment:
Need a return type to pass the tests.

Reply:
I reverted to PROFILE_MAP = {'generic_delete': generic_delete, ...} so the function does not exist any more.